Cargo.toml

```diff
@@ -70,4 +70,4 @@ bitflags = "2"
 tracing = "0.1.37"
 fnv = "1.0.7"
 instant = { version = "0.1.6", features = ["wasm-bindgen"] }
-winit = "0.29"
+winit = { version = "0.30", features = ["rwh_05"]}
```

Comment:
This `rwh_05` isn't needed.

Reply:
Ope, yeah had to add that to figure something out, but not needed now.